confgenerator/files.go

```diff
@@ -52,6 +52,10 @@ func (uc *UnifiedConfig) GenerateFilesFromConfig(ctx context.Context, service, l
 	case "": // Validate-only.
 		return nil
 	case "fluentbit":
+		// short circuit in case of BMS
+		if platform.FromContext(ctx).ResourceOverride != nil {
```

Comment:
This will catch other resource overrides, too. (and I assume this is a TODO, right? you're going to eventually support logging?)

Please add an explicit `SkipLogging` bool to the Platform struct, that you can set on BMS.

Reply:
Done.

Reply:
Yes, I'll add a support for logging later.